homeassistant/components/mullvad/binary_sensor.py

```diff
@@ -0,0 +1,55 @@
+"""Setup Mullvad VPN Binary Sensors."""
+from homeassistant.components.binary_sensor import (
+    DEVICE_CLASS_CONNECTIVITY,
+    BinarySensorEntity,
+)
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from .const import DOMAIN
+
+BINARY_SENSORS = (
+    {
+        "id": "mullvad_exit_ip",
+        "name": "Mullvad Exit IP",
+        "device_class": DEVICE_CLASS_CONNECTIVITY,
+    },
+)
+
+
+async def async_setup_entry(hass, config_entry, async_add_entities):
+    """Defer sensor setup to the shared sensor module."""
+    coordinator = hass.data[DOMAIN]
+
+    async_add_entities(
+        MullvadBinarySensor(coordinator, sensor) for sensor in BINARY_SENSORS
+    )
+
+
+class MullvadBinarySensor(CoordinatorEntity, BinarySensorEntity):
+    """Represents a Mullvad binary sensor."""
+
+    def __init__(self, coordinator, sensor):  # pylint: disable=super-init-not-called
+        """Initialize the Mullvad binary sensor."""
+        self.coordinator = coordinator
+        self.id = sensor["id"]
+        self._name = sensor["name"]
```

Comment:
Use constants as well. The device class was passed and part of the binary sensor dictionary but not used.

```suggestion
        self.id = sensor[CONF_ID]
        self._name = sensor[CONF_NAME]
        self._device_class = sensor[CONF_DEVICE_CLASS]
```